src/ScottPlot5/ScottPlot5/ScopedTransform.cs

```diff
@@ -0,0 +1,26 @@
+﻿using SkiaSharp;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ScottPlot
+{
+    internal class ScopedTransform : IDisposable // TODO: I'm a little uneasy on this name?
```

Comment:
I don't mind the name. As long as the XML docs make it clear what it does, it should be fine. I would consider moving tools like this into a `Drawing` namespace though. I imagine we will start collecting stuff like this (e.g., font stuff).

---

I see there's already a `Drawing.cs`, but it looks pretty small, so I'm thinking that should get broken apart and moved into better-named classes in the new drawing namespace too. After doing this we could have something like:

* `ScottPlot.Drawing.ScopedTransform`
* `ScottPlot.Drawing.Text.MeasureString()`
* `ScottPlot.Drawing.Text.MeasureLargestString()`
* `ScottPlot.Drawing.Shapes.DrawLines()`

It's probably not too big a change to go ahead and do in this PR, but if it gets more complex than expected we can open an issue so we don't forget and address it after it's merged

... also I'm not 100% those methods are even being called anymore, so if they're not being used we can just delete them

Reply:
I think XML docs are unnecessary, as this is `internal`. I do like the idea of a drawing namespace but that might be better handled in a separate PR.

> also I'm not 100% those methods are even being called anymore
Can't speak for all of them, but in this PR I'm using `MeasureString`, I actually had to modify it because it wasn't quite correct.